runtime/compiler/optimizer/InlinerTempForJ9.cpp

```diff
@@ -3047,15 +3044,12 @@ TR_J9InlinerPolicy::adjustFanInSizeInExceedsSizeThreshold(int bytecodeSize,
    static const char *q = feGetEnv("TR_SizeMultiplier");
    static const uint32_t multiplier = ( q ) ? atoi (q) : SIZE_MULTIPLIER;
 
-   static const char *qq = feGetEnv("TR_Min_FanIn_Size");
-   static const uint32_t min_size = ( qq ) ? atoi(qq) : MIN_FAN_IN_SIZE;
```

Comment:
Probably a good idea to delete the macro definition of `MIN_FAN_IN_SIZE`; it looks to be defined in two different files and (after this PR) it's only referenced in a comment.

Reply:
Good point. Done.